iphone/cli/hooks/package.js

```diff
@@ -112,31 +112,6 @@ exports.init = function (logger, config, cli) {
 						}
 					});
 
-					var now = new Date;
-					var destInfoPlist = path.join(stagingArchiveDir, 'Info.plist');
-					if (!fs.existsSync(destInfoPlist)) {
-						var origPlist = new appc.plist(path.join(builder.buildDir, 'Info.plist'));
-						var newPlist = new appc.plist();
-						var appBundle = 'Applications/' + name + '.app';
-
-						appc.util.mix(newPlist, {
-							ApplicationProperties: {
-								ApplicationPath: appBundle,
-								CFBundleIdentifier: origPlist.CFBundleIdentifier || builder.tiapp.id,
-								CFBundleShortVersionString: origPlist.CFBundleShortVersionString || '1.0',
-								CFBundleVersion: origPlist.CFBundleVersion || '1.0',
-								SigningIdentity: 'iPhone Distribution: ' + builder.certDistributionName,
-								IconPaths: [
-									appBundle + '/' + builder.tiapp.icon
-								]
-							},
-							ArchiveVersion: newPlist.type('integer', 2),
-							CreationDate: now,
-							Name: name,
-							SchemeName: name
-						}).save(destInfoPlist);
-					}
```

Comment:
Um, no, we need the Info.plist. I had erroneously had the same code twice and thus removed the duplicate. It should run for both dist-appstore and dist-adhoc builds.